lib/config.js

```diff
@@ -202,12 +202,25 @@ cc('veres-one.config', () => ({
   consensusMethod: 'Continuity2017',
   electorSelectionMethod: {
     type: 'VeresOne',
+    // FIXME: how is this generated?
+    electorPool: 'urn:uuid:a3dd75aa-bb78-431d-b767-630831882545',
   },
-  // FIXME: add config validator
-  // ledgerConfigurationValidator: [
-  //
-  // ],
-  // FIXME: Enable validation
+  ledgerConfigurationValidator: [{
+    type: 'VeresOneValidator2017',
+  }, /*{
+    FIXME: validator can't validate sigs using DID keys, how is that supposed
+    to work?  Maintainer DID is generated in ledger.js and the key associated
+    with that is used to sign configuration.  Signature Validator attempts to
+    dereference the key (e.g. did:v1:test:nym:z2cJ...Kx7#authn-key-1) and fails.
```

Comment:
Why does it fail? (Like, what's the error etc)

Reply:
The DIDs being used are not on the ledger yet.  So presumably the public keys cannot be dereferenced.

Reply:
And they are not on the ledger yet because the ledger does not exist yet, this is the genesis configuration.

Reply:
Ahhh right makes sense.